app/src/main/AndroidManifest.xml

```diff
@@ -16,7 +16,8 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         tools:replace="android:icon,android:theme">
-        <activity android:name=".service_locator.EntryActivity">
+        <activity android:name=".service_locator.EntryActivity"
+            android:noHistory="true" >
```

Comment:
When you press the android back button in the main menu, you go back to the 'EntryActivity', only that activity is not used. This prevents android from putting it on the stack of last activities